src/main/webapp/index.html

```diff
@@ -5,13 +5,24 @@
     <title>Capstone Project</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="nutrition.js"></script>
```

Comment:
It is not best practice to use multiple separate JavaScript files as it increases the loading time of a page. Wherever possible you usually only want to have one. See this for further reading https://developer.yahoo.com/performance/rules.html?guccounter=1

Reply:
+1, but to be precise this doesn't mean project developers implement Javascript code in one single file. 

The common practice is to develop the code in separate files, and minify all files into one file at the deployment phase :) Of course this is an optimization for large size production projects and it's an overkill for our capstone project.